src/stateMachines/unconfiguration.ts

```diff
@@ -92,12 +92,45 @@ export class Unconfiguration {
               message: (context, event) => event.data,
               clientId: (context, event) => context.clientId
             },
-            onDone: 'REMOVE_REMOTE_ACCESS_POLICY_RULE_USER_INITIATED'
+            onDone: 'GET_8021X_PROFILE'
           },
           on: {
             ONFAILED: 'FAILURE'
           }
         },
+        GET_8021X_PROFILE: {
+          invoke: {
+            src: this.get8021xProfile.bind(this),
+            id: 'get-8021x-profile',
+            onDone: {
+              actions: assign({ message: (context, event) => event.data }),
+              target: 'CHECK_GET_8021X_PROFILE_RESPONSE'
+            },
+            onError: {
+              target: 'ERROR'
+            }
+          }
+        },
+        CHECK_GET_8021X_PROFILE_RESPONSE: {
+          always: [
+            {
+              cond: 'is801xProfileEnabled',
```

Comment:
name is missing '2' -> is8021xProfileEnabled

Reply:
resolved